README.md

```diff
@@ -156,11 +158,12 @@ between client-python versions.
 | 26.0 Alpha/Beta | Kubernetes main repo, 1.26 branch    | ✗                             |
 | 26.0            | Kubernetes main repo, 1.26 branch    | ✗                             |
 | 27.0 Alpha/Beta | Kubernetes main repo, 1.27 branch    | ✗                             |
-| 27.0            | Kubernetes main repo, 1.27 branch    | ✓                             |
+| 27.0            | Kubernetes main repo, 1.27 branch    | ✗                             |
```

Comment:
Nit: According to the [doc](https://github.com/kubernetes-client/python#:~:text=support%20three%20GA%20major%20releases) we will drop 27.0 support when 30.0 GA comes out.

Reply:
Fixed